onnxruntime/python/tools/quantization/matmul_4bits_quantizer.py

```diff
@@ -578,6 +812,8 @@ def __init__(
         accuracy_level: int | None = None,
         nodes_to_exclude=None,
         quant_format=QuantFormat.QOperator,
+        ops_to_quantize: tuple[str, ...] | None = None,
```

Comment:
Not all the Gather need to be quantized. we only quantizes the specified Gather.